fem/bilininteg.cpp

```diff
@@ -3458,17 +3464,7 @@ void DGDiffusionIntegrator::AssembleFaceMatrix(
    const IntegrationRule *ir = IntRule;
    if (ir == NULL)
    {
-      // a simple choice for the integration order; is this OK?
-      int order;
-      if (ndof2)
-      {
-         order = 2*max(el1.GetOrder(), el2.GetOrder());
-      }
-      else
-      {
-         order = 2*el1.GetOrder();
-      }
-      ir = &IntRules.Get(Trans.GetGeometryType(), order);
+      ir = &GetRule(el1.GetOrder(), Trans);
```

Comment:
Is there an easy way to preserve this behavior?

Reply:
You mean using the max polynomial degree of neighboring elements?

Reply:
Indeed. I think keeping the maximum polynomial degree per default might still be of interest for p and hp adaptivity (in the future).

Reply:
Yes, it should be easy to replace `el1.GetOrder()` with the max (if there is a second element, following the logic of the deleted code). cc: @arotem3